nodes/bm_inventory_r4pac10.json

```diff
@@ -1011,9 +1011,10 @@
         {
             "name": "MOC-R4PAC10U29-S1",
             "properties": {
-                "capabilities": "iscsi_boot:True"
+                "capabilities": "iscsi_boot:True",
+                "root_device": "{\"wwn\": \"0x5002538a983785a0\"}"
```

Comment:
I think you want this to be a nested dictionary; otherwise you end up with a node like this that gets the error "No disks could be found using the root device hints"

```
properties             | {'capabilities': 'iscsi_boot:True', 'root_device': '{"wwn": "0x5002538a4910ece0"}', 'vendor': 'dell inc'}   
```

Reply:
Isn't it already a nested dict? Maybe I'm not understanding what you mean, this is what it currently is:

```
"properties": {
    "capabilities": "iscsi_boot:True",
    "root_device": "{\"wwn\": \"0x5002538a4910ece0\"}"
},
```

Reply:
You want something more like:

```
"properties": {
    "capabilities": "iscsi_boot:True",
    "root_device": {"wwn": "0x5002538a4910ece0"}
},
```

What you have right now is being interpreted as a string.